README.md

```diff
@@ -57,17 +70,6 @@ The instructor will then review the pull request and make comments should furthe
 
 ## Chapter 12 Questions
 
```

Comment:
Please merge the Course Wiki into your fork in order to avoid accidental deletions of existing content.